errors/Cargo.toml

```diff
@@ -14,5 +14,5 @@ bevy = { path = ".." }
 workspace = true
 
 [package.metadata.docs.rs]
-rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
+rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs", "--generate-link-to-definition"]
```

Comment:
Small review note: I added the argument to these crates as well for consistency, although they don't appear to be documented on docs.rs